cartographer_grpc/framework/rpc.h

```diff
@@ -39,17 +40,27 @@ class Rpc {
     const State state;
     Service* service;
     Rpc* rpc;
+    bool pending;
   };
 
   Rpc(int method_index, ::grpc::ServerCompletionQueue* server_completion_queue,
       const RpcHandlerInfo& rpc_handler_info, Service* service);
 
+  void OnRequest();
+  void OnReadsDone();
+  void Write(std::unique_ptr<::google::protobuf::Message> message);
+
   int method_index() const { return method_index_; }
   ::grpc::ServerCompletionQueue* server_completion_queue();
   ::grpc::internal::RpcMethod::RpcType rpc_type() const;
   ::grpc::ServerContext* server_context() { return &server_context_; }
   ::grpc::internal::ServerAsyncStreamingInterface* streaming_interface();
-  RpcState* GetRpcState(State state);
+  ::grpc::internal::AsyncReaderInterface<::google::protobuf::Message>*
+  async_reader_interface();
+
+  RpcState* SetRpcStatePending(State state, bool pending);
```

Comment:
Rpc is already the name of the class so it can be removed from the method name, also in the next line.